### ISSUE TITLE
脱ブラックボックスコース > ネットワークの訓練

### ISSUE THREAD

**Issue by mitani1012** [kikagaku/kikagaku-public-issues#57]
テストQ1についての質問です。
train_loaderはdrop = True, shuffle = Trueを行いましたが、
その条件での正解率を求めれば良いのでしょうか？

Q2, Q3は正解するも、Q1が何度試行しても正解せず不安に感じております。
過去に同様な質問をされた方への返信も参考に自分のコードを見直していますが、シードの固定等に問題はありません。

340個の中での正答率か341個の中の正答率か、
訓練済みのデータを用いて計算するのか、訓練中の正答率を答えるのかを教えていただけると幸いです。

どうぞよろしくお願いいたします

**Comment by nakajima0121** [comment]
私はQ2だけ正解したのですが、Q1/Q3が正解できません。Q2/Q3は shuffle = false のため、何度行なっても結果は同じになるのですが、Q2だけ正解するという理由もよくわからず、何時間も前に進めず困っております。

**Comment by mitani1012** [comment]
お世話になっております。
こちらは回答にお時間がかかっていると考えてよろしいでしょうか。
忘れられている可能性を考えこちらに新たにコメントさせていただきます。

**Comment by 0browns16** [comment]
私もQ2, Q3は正解を導くここができても、Q1だけは何度試行しても正解できません。

**Comment by nakabeppu** [comment]
ご連絡が遅くなり大変申し訳ございません。
テストの要件を追加いたしましたので、再度挑戦していただけますと幸いです。
